File: CompetitiveProgramming/CCC/CCC '00 J2 - 9966/2019-03-21/Main.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main
{
	static boolean flippable (int j)
	{
		String num = Integer.toString(j);
		for(int i = 0; i < num.length(); i++)
		{
			switch (num.charAt(i))
			{
				case '1': if(num.charAt(num.length()-1-i) != '1') return false; break;
				case '8': if(num.charAt(num.length()-1-i) != '8') return false; break;
				case '0': if(num.charAt(num.length()-1-i) != '0') return false; break;
				case '6': if(num.charAt(num.length()-1-i) != '9') return false; break;
				case '9': if(num.charAt(num.length()-1-i) != '6') return false; break;
				default: return false;
			}
		}
		return true;
	}
	
	public static void main (String [] args) throws IOException
	{
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		int lower = Integer.parseInt(br.readLine());
		int higher = Integer.parseInt(br.readLine());
		
		int count = 0;
		for(int i = lower; i <= higher; i++)
			if(flippable(i))
				count++;
		System.out.println(count);
	}
}

File: CompetitiveProgramming/CCC/CCC '00 J2 - 9966/2019-03-21/Main.cpp
#include <iostream>
using namespace std;

bool flippable (int j)
{
	string num = to_string(j);
	for(int i = 0; i < num.length(); i++)
	{
		switch (num[i])
		{
			case '1': if(num[num.length()-1-i] != '1') return false; break;
			case '8': if(num[num.length()-1-i] != '8') return false; break;
			case '0': if(num[num.length()-1-i] != '0') return false; break;
			case '6': if(num[num.length()-1-i] != '9') return false; break;
			case '9': if(num[num.length()-1-i] != '6') return false; break;
			default: return false;
		}
	}
	return true;
}

int main()
{
	int lower, higher;
	cin >> lower >> higher;
	
	int count = 0;
	for(int i = lower; i <= higher; i++)
		if(flippable(i))
			count++;
	cout << count;
	
	return 0;
}